src/controllers/users.js

```diff
@@ -0,0 +1,82 @@
+'use strict';
+
+const userService = require('../services/users');
+
+const getAll = (req, res) => {
+  const users = userService.getAll();
+
+  res.send(users);
+};
+
+const getOne = (req, res) => {
+  const { userId } = req.params;
+  const foundedUser = userService.getById(userId);
```

Comment:
The userService.getById() function is called without await, which can lead to an unhandled promise rejection if the promise is rejected. You should use async/await or promise chaining to handle the promise properly.